rllib/agents/ppo/ddppo.py

```diff
@@ -131,6 +136,15 @@ def validate_config(config):
         raise ValueError(
             "Distributed data parallel requires truncate_episodes "
             "batch mode.")
+    if config["kl_coeff"] != 0.0 or config["kl_target"] != 0.0:
+        """DDPPO doesn't support KL penalties like PPO-1. 
```

Comment:
Sorry, nit-picking here :)
Can we make all comments as one-liners on top of the code block that they refer to?
The only situation where we should use `"""`-style comments is for our classes/methods/functions' docstrings.

So this should be something like:
```
# DDPPO doesn't support KL penalties like PPO-1.
# In order to support KL penalties, DDPPO would need to 
# become undecentralized, which defeats the purpose of the 
# algorithm. Users can still tune the entropy coefficient to 
# control the policy entropy (similar to controlling the KL 
# penalty).
if config["kl_coeff"] != 0.0 or config["kl_target"] != 0.0:
...
```

Reply:
gotcha

Reply:
fixed!